exporter/elasticsearchexporter/integrationtest/exporter_bench_test.go

```diff
@@ -101,19 +106,119 @@ func benchmarkTraces(b *testing.B, batchSize int, mappingMode string) {
 		require.NoError(b, exporter.ConsumeTraces(ctx, traces))
 		b.StopTimer()
 	}
-	b.ReportMetric(
-		float64(runnerCfg.generatedCount.Load())/b.Elapsed().Seconds(),
-		"events/s",
+	require.NoError(b, exporter.Shutdown(ctx))
+	reportMetrics(b, runnerCfg)
+}
+
+// BenchmarkExporterFlushTimeout benchmarks exporter flush triggered by "flush timeout" aka flush interval.
+func BenchmarkExporterFlushTimeout(b *testing.B) {
```

Comment:
[to reviewer] this actually gave me OOM when I run locally, and I incline to remove this bench

Reply:
I'd drop it. I'm not sure this one is particularly beneficial anyway. You would likely still want to have a size threshold, and have the timeout in case throughput * interval is below that limit. I think we mostly care about benchmarking the high throughput case.

Reply:
> have the timeout in case throughput * interval is below that limit.

In that case, the bench will effectively be throttled for flush_timeout for every Consume* call because Consume* call will block until flush. This particular bench that I'm planning to remove worked around this by using RunParallel.

@lahsivjar and I spent some time on this and we thought that going forward the new `BenchmarkExporterFlushItems` will be more useful for comparing across versions.

Reply:
> we thought that going forward the new BenchmarkExporterFlushItems will be more useful for comparing across versions

++